torch/csrc/jit/codegen/cuda/index_compute.cpp

```diff
@@ -766,7 +767,10 @@ kir::TensorIndex* Index::getGlobalProducerIndex(
   // Replay producer to look like consumer so we can index on producer since our
   // loop nests look like consumer
   auto producerAsC = TransformReplay::replayPasC(
-                         producer_tv->domain(), consumer_tv->domain(), -1)
+                         producer_tv->domain(),
```

Comment:
can we replace this verbose parameter list with a struct?

Reply:
We could, though not sure how much it would help.